index.htm

```diff
@@ -122,6 +126,39 @@ <h2>We Make Your Home Alive</h2>
 Begonia
 </div>
 
+<h1>About WaterMi</h1>
+
+    <p> 
+    </p>
+
+  <h2>Values</h2>
+      <p>
+        <ul>
+          <li>Value 1</li>
+          <dd>&#9900; Lorem 1 ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.</dd>
+          <li>Value 2</li>
+          <dd>&#9900; Lorem 2 ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.</dd>
+          <li>Value 3</li>
+          <dd>&#9900; Lorem 3 ipsum dolor sit amet, consetetur sadipscing elitr, sed diam.</dd>
+        </ul>
+      </p>  
+
+  <p>
+  <img src="./res/img/team.jpg" alt="WaterMi Staff" id="imgStaff">
```

Comment:
why is this img in a p element? p is for paragraphs

Reply:
I put it inside a p element because I was trying to center the image horizontally.  I couldn't figure out how to do it any other way.